dopamine/discrete_domains/train.py

```diff
@@ -51,8 +50,12 @@ def main(unused_argv):
     unused_argv: Arguments (unused).
   """
   tf.logging.set_verbosity(tf.logging.INFO)
-  run_experiment.load_gin_configs(FLAGS.gin_files, FLAGS.gin_bindings)
-  runner = run_experiment.create_runner(FLAGS.base_dir)
+  if FLAGS.async:
+    runner = run_experiment.create_runner(
+        FLAGS.base_dir, schedule='async_train')
+  else:
+    run_experiment.load_gin_configs(FLAGS.gin_files, FLAGS.gin_bindings)
```

Comment:
Is this intentionally behind the `else`?  If so, I'm a little confused about how the trainer is set up.

Reply:
Good catch. This was intentional because async runner doesn't handle gin for now.

I am fixing this in my last PR. Are you fine ignoring this until then?

Reply:
Added a TODO(#124) to track this.